includes/class-wpm-posts.php

```diff
@@ -41,7 +41,7 @@ public function __construct() {
 		add_filter( 'the_post', 'wpm_translate_post', 5 );
 		add_filter( 'the_title', 'wpm_translate_string', 5 );
 		add_filter( 'the_content', 'wpm_translate_string', 5 );
-		add_filter( 'the_excerpt', 'wpm_translate_string', 5 );
+		add_filter( 'get_the_excerpt', 'wpm_translate_string', 5 );
```

Comment:
Not everywhere the filter "get_the_excerpt" is using. As example https://i.imgur.com/HZbHs87.png